tests/test_cases/input/slashdash_escline_before_node.kdl

```diff
@@ -0,0 +1,3 @@
+/-\
```

Comment:
Based on the way ckdl failed this (now fixed), is it possible this is the only test for a line continuation outside of a node?

Reply:
oh it might be. Although I _feel_ like I saw at least one that had leading esclines before the first node. Would be nice to have plain escline tests with nothing else.